examples/add_sub/model.py

```diff
@@ -38,6 +38,50 @@ class TritonPythonModel:
     that is created must have "TritonPythonModel" as the class name.
     """
 
+    def auto_complete_config(self, args):
+        """`auto_complete_config` is called only once when the server is started
+        with `--strict-model-config=false`. Implementing `auto_complete_config`
+        function is required when using auto-complete for the model configuration.
+        This function allows us to set `max_batch_size`, `input` and `output`
+        properties of the model using `pb_utils.set_max_batch_size`,
+        `pb_utils.set_input`, and `pb_utils.set_output`.
+        There are three objects for setting the model configuration for
+        auto-complete:
+          `pb_utils.InferInputConfig`: describe input configuration
+          `pb_utils.InferOutputConfig`: describe output configuration
+          `pb_utils.ModelConfig`: describe the model configuration
+        Must return a `pb_utils.ModelConfig` object which contains the updated
+        model configuration.
+        Note: All the objects in this function will go out of scope after exiting.
+        Should not store any objects in this function.
+
+        Parameters
+        ----------
+        args : dict
+          Both keys and values are strings. The dictionary keys and values are:
+          * model_config: A JSON string containing the current model configuration
```

Comment:
How about constructing a `ModelConfig` object instead of JSON and passing it to the `auto_complete_config` here?

Reply:
May I ask is it because we want to pass the JSON with a wrapper as well?

Reply:
It is not because of that. I think we should be consistent in using the abstractions and that is why it might be better to wrap it in that object.